Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/TapToPlace/TapToPlace.cs

```diff
@@ -0,0 +1,467 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Physics;
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Utilities.Solvers;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Utilities
+{
+    /// <summary>
+    /// Tap to place is a far interaction component used to place objects on a surface.
+    /// </summary>
+    public class TapToPlace : Solver, IMixedRealityPointerHandler
+    {
+        [Experimental]
+        [SerializeField]
+        [Tooltip("The game object that will be placed if it is selected")]
+        private GameObject gameObjectToPlace;
+
+        /// <summary>
+        /// The game object that will be placed if it is selected
+        /// </summary>
+        public GameObject GameObjectToPlace
+        {
+            get => gameObjectToPlace;
+            set
+            {
+                if (value != null && gameObjectToPlace != value)
```

Comment:
Is it not a valid transition to clear the gameObjectToPlace? i.e. if i wanted to make a game object not the one to place, the only way for me to do that with this code is to point it at something else, and then I guess delete that other dummy object.

Reply:
I removed the `GameObjectToPlace` property because the game object to place will always be the `gameObject` TapToPlace is attached to.